mindmeld/query_cache.py

```diff
@@ -53,7 +58,7 @@ def __init__(self, app_path):
 
         cursor = self.disk_connection.cursor()
 
-        if not self.compatible_version():
+        if not self.compatible_version() or not self.compatible_text_pipeline_config():
```

Comment:
`compatible_text_pipeline_config ` can be consolidated into compatible_version by doing the following SQL query:
```
        cursor = self.disk_connection.cursor()
        try:
            cursor.execute("""
            SELECT version_number FROM version WHERE version_number=(?) and text_pipeline_hash=(?);
            """, (ProcessedQuery.version,self.text_pipeline_hash))
            if len(cursor.fetchall()) == 0:
                # version does not match
                return False
            return True
        except Exception:  # pylint: disable=broad-except
            return False
```

Reply:
Removed the dependence on an external ProcessedQuery version number.